Bluepill-runner/Bluepill-runner/BPPacker.m

```diff
@@ -164,20 +218,30 @@ @implementation BPPacker
     }]];
     NSNumberFormatter *fmt = [[NSNumberFormatter alloc] init];
     [fmt setPositiveFormat:@".02"];
+    int bpNum = 0;
     for (BPXCTestFile *bundle in sortedBundles) {
-        [BPUtils printInfo:INFO withString:@"Estimated time is %@ for %@", [fmt stringFromNumber:bundle.estimatedExecutionTime], bundle.testBundlePath];
+        [BPUtils printInfo:INFO withString:@"[BP-%d] %@ estimated to take %@ seconds and skips %lu out of %lu tests.", ++bpNum, bundle.name, [fmt stringFromNumber:bundle.estimatedExecutionTime], (unsigned long)[bundle.skipTestIdentifiers count], (unsigned long)[bundle.allTestCases count]];
     }
     return sortedBundles;
 }
```

Comment:
This above function seems too huge and difficult to read and understand. Can you divide it into smaller parts and maybe add the general splitting/packing strategy as comment at the top of this function?